vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/format/NoticeEventFormatter.kt

```diff
@@ -317,4 +317,28 @@ class NoticeEventFormatter @Inject constructor(private val sessionHolder: Active
             else                 -> null
         }
     }
+
+    fun getRedactionReason(event: Event): String {
```

Comment:
Maybe rename the method to `formatRedactedEvent` to respect the naming in this class